src/components/feedback/snackbar/Snackbar.stories.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useState } from 'react';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { expect, within, userEvent } from '@storybook/test';
+
+import { Snackbar, SnackbarProps } from './Snackbar';
+import { Button } from '../../basic/Button';
+
+const meta: Meta = {
+	title: 'Components/Feedback/Snackbar',
+	component: Snackbar,
+	parameters: {
+		layout: 'padded'
+	}
+} satisfies Meta<typeof Snackbar>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const SnackbarStory = ({ open, label, ...rest }: SnackbarProps): React.JSX.Element => {
+	const [snack, setSnack] = useState(false);
+
+	return (
+		<>
+			<Button type="outlined" color="info" label="Click" onClick={() => setSnack(true)} />
+			<Snackbar onClose={() => setSnack(false)} label={label} open={open || snack} {...rest} />
+		</>
+	);
+};
+
+export const Success: Story = {
+	args: {
+		label: 'Lorem Ipsum dolor sit amet',
+		type: 'success',
+		open: false,
+		actionLabel: 'Ok',
+		disableAutoHide: false,
+		hideButton: false,
+		zIndex: 1000,
+		autoHideTimeout: 4000,
+		disablePortal: false,
+		singleLine: false
+	},
+	render: ({ open, label, ...rest }) => <SnackbarStory label={label} open={open} {...rest} />,
+	play: async ({ canvasElement }) => {
+		const canvas = within(canvasElement);
+		const { body } = canvasElement.ownerDocument;
+		const button = canvas.getByRole('button', {
+			name: /success/i
```

Comment:
This is not working anymore since the button has the name click for every story.